content/en/docs/contribute/style/hugo-shortcodes/index.md

```diff
@@ -24,7 +24,7 @@ Below is a demo of the feature state snippet, which displays the feature as stab
 {{</* feature-state for_k8s_version="v1.10" state="stable" */>}}
 ```
 
-Will render to: 
+It renders to: 
```

Comment:
Make consistent with the usage you change further down the page:
s/It renders/Renders

Reply:
Ack! Thank you!